client/blocks/reader-post-card/featured-video.jsx

```diff
@@ -15,7 +15,7 @@ import FeaturedImage from './featured-image';
 class FeaturedVideo extends React.Component {
 
 	static propTypes = {
-		thumbnailUrl: React.PropTypes.string,
+		thumbnailUrlPromise: React.PropTypes.object,
```

Comment:
Let's avoid doing this if we can and pass through thumbnailUrl and preferThumbnail. We can either connect the component or make sure the caller is smart enough to pass through the right information